electron_app/src/DBManager.js

```diff
@@ -105,6 +107,16 @@ const createContactsIfOrNotStore = async (contacts, trx) => {
   return emailAddresses;
 };
 
+const updateContactScore = (emailId, trx) => {
+  return trx.raw(
+    `UPDATE ${
+      Table.CONTACT
+    } SET SCORE = SCORE + 1 WHERE id IN (SELECT contactId from ${
```

Comment:
You can use `knex.whereIn()`, `knex.increment()` and raw query only on where clausule (or another knex method query)